src/Workspaces/Core/Portable/Shared/Extensions/ICompilationExtensions.cs

```diff
@@ -138,5 +138,10 @@ public static INamedTypeSymbol SuppressMessageAttributeType(this Compilation com
         {
             return compilation.GetTypeByMetadataName("System.Diagnostics.CodeAnalysis.SuppressMessageAttribute");
         }
+
+        public static INamedTypeSymbol TupleElementNamesAttributeType(this Compilation compilation)
+        {
+            return compilation.GetTypeByMetadataName("System.Runtime.CompilerServices.TupleElementNamesAttribute");
```

Comment:
Yes, actually it is `null` in test were I don't provide an implementation of `TupleElementNamesAttribute`, so I ran into this ;-)